FlowCrypt/src/main/java/com/flowcrypt/email/api/retrofit/response/model/node/NodeKeyDetails.kt

```diff
@@ -37,7 +40,8 @@ data class NodeKeyDetails constructor(@Expose val isFullyDecrypted: Boolean?,
                                       @Expose val expiration: Long,
                                       @Expose val algo: Algo?,
                                       var passphrase: String?,
-                                      var errorMsg: String?) : Parcelable {
+                                      var errorMsg: String?,
+                                      val keyRing: PGPKeyRing? = null) : Parcelable {
```

Comment:
@IvanPizhenko Please note if you change a constructor of `data` class that implements `Parcelable` you should modify the implementation of `Parcelable` methods. Otherwise, you will receive a runtime error. It's Android thing. I hope in the future after https://github.com/FlowCrypt/flowcrypt-android/issues/1034 we will have not to do that.

Reply:
@IvanPizhenko Please don't try to fix that for now. It seems I need to make some code changes. Too many Android things.

Reply:
well, I didn't try to fix anything